packages/react-storage/src/components/StorageBrowser/composables/__tests__/DataTable/dataCells/ButtonDataCell.spec.tsx

```diff
@@ -32,6 +32,25 @@ describe('ButtonDataCell', () => {
     expect(svg).toBeInTheDocument();
   });
 
+  it('renders button with aria-label', () => {
+    const { container } = render(
+      <ButtonDataCell content={{ icon: 'cancel', ariaLabel: 'label' }} />
+    );
+
+    const buttonDataCell = screen.getByRole('button');
```

Comment:
Can destructure `getByRole` from the return of `render`

Reply:
fixed